src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  position: relative;
+  max-width: 708px;
```

Comment:
The max-width should be calculated based on the block size, gap, and paddings to make it more flexible.